src/vectorchord/advanced-features/pre-and-post-filtering.md

```diff
@@ -0,0 +1,57 @@
+# Prefiltering And Postfiltering <badge type="tip" text="since v0.4.0" />
+
+VectorChord provides flexible filtering mechanisms to improve vector search performance and accuracy through **prefiltering** and **postfiltering** strategies. These filtering approaches determine when query conditions (like `WHERE` clauses) are applied during the vector similarity search process.
+
+Unlike **postfiltering**, which is the default behavior, the **prefiltering** strategy applies an additional filter before the vector search. This prunes the search space, thereby increasing the speed of the search. However, the additional filter also introduces an extra cost that should be considered.
```

Comment:
Unless it's my hallucination, I’ve already emphasized this here as well: don’t frame the two filtering methods as being in opposition. You should tell the user that prefiltering is essentially performing pruning.